src/libraries/Microsoft.Bcl.Numerics/tests/MathF.cs

```diff
@@ -0,0 +1,1151 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+
+#pragma warning disable xUnit1025 // reporting duplicate test cases due to not distinguishing 0.0 from -0.0
+
```

Comment:
Same here, it'd be nice to callout the explicit differences between the .NET Core and .NET Framework tests at the top here.

Reply:
Since .NET Core is 100% a forward I have no tests running on .NET Core. Do you still want me to add a commetn like that even though we don't run those tests there?

Reply:
I think it's worth calling out the differences for clarity. That way we can, at a glance, get a basic gist of what is different in the tests.